packages/playground/src/sidebar/runtime.ts

```diff
@@ -94,19 +91,15 @@ function rewireLoggingToElement(
       console.error(i("play_run_js_fail"))
       console.error(error)
     }
-    curLog++
   })
 
-  function bindLoggingFunc(obj: any, raw: any, name: string, id: string, cur: number) {
+  function bindLoggingFunc(obj: any, raw: any, name: string, id: string) {
     obj[name] = function (...objs: any[]) {
       const output = produceOutput(objs)
       const eleLog = eleLocator()
       const prefix = `[<span class="log-${name}">${id}</span>]: `
       const eleContainerLog = eleOverflowLocator()
-      const index = cur - offset
-      if (index >= 0) {
-        allLogs[index] = (allLogs[index] ?? '') + prefix + output + "<br>"
-      }
+      allLogs.push(prefix + output + "<br>");
```

Comment:
This could be represented as
```suggestion
      allLogs.push(`${prefix}${output}<br>`);
```